Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
+=== Instantiating JDG Online Service
 
-The `modules` directory contains modules used in `image.yaml`. See http://concreate.readthedocs.io/en/develop/[Concreate] documentation for more details.
+Once the above templates are correctly installed, one can instantiate them using proper icon in the catalog.
 
-The `functional-tests` directory contains http://arquillian.org/arquillian-cube/[Arquillian Cube tests]. The tests assume that there's a running OpenShift cluster and `oc` or `kubectl` is already attached to it.
+image::documentation/img/catalog-walkthrough.gif[JDG Online Service in the Service Catalog]
 
-How can I play with it?
------------------------
+Both Caching and Shared Memory services define a list of optional and required parameters, which have been explained below:
 
-At first you will need a working OpenShift cluster as well as `oc` client in your `$PATH`. See https://openshift.io/[OpenShift documentation] how to achieve this.
+.Caching Service parameters
+[options="header"]
+|======================
+|Option                    |Is required? |Description                                                                              |Default option
+|NAMESPACE                 |Yes          |Namespace in which this service was created. Used for setting up clustering configuration.|myproject
+|APPLICATION_NAME          |Yes          |Application name                                                                         |caching-service-app
+|IMAGE                     |Yes          |Image that will be used to bootstrap the service                                         |jboss-dataservices/datagrid-online-services
+|NUMBER_OF_INSTANCES       |Yes          |Number of instances                                                                      |1
+|TOTAL_CONTAINER_MEM       |No           |Total container size (both limits and requests)                                          |512
+|APPLICATION_USER          |Yes          |Username used for authentication                                                         |<none>
+|APPLICATION_USER_PASSWORD |No           |Password used for authentication                                                         |<generated>
+|======================
 
-The next step is to install the templates. Feel free to use our `Makefile` or manually invoke `oc create -f templates/caching-service.json -n openshift`. After this step you should see the new service in OpenShift UI.
+.Shared Memory Service parameters
+[options="header"]
+|======================
+|Option                    |Is required? |Description                                                                              |Default option
+|NAMESPACE                 |Yes          |Namespace in which this service was created. Used for setting up clustering configuration.|myproject
+|APPLICATION_NAME          |Yes          |Application name                                                                         |caching-service-app
```

Comment:
This one should be shared-memory-service-app

Reply:
Done